Header header
geometry_msgs/Point point
float32 confidence